src/modules/core/components/ProgressBar/ProgressBar.tsx

```diff
@@ -42,12 +42,13 @@ const ProgressBar = ({
 }: Props) => {
   const { formatMessage } = useIntl();
   const titleText = formatMessage(MSG.titleProgress, { value, max });
+
```

Comment:
So the `value` for the `ProgressBar` does not get loaded/fetched correctly.

Please open a issue to track investigating that

Reply:
Sure thing, I can do that!